src/services/billing/cost-management/widgets/SpcProjectWiseUsageSummary.vue

```diff
@@ -98,19 +203,43 @@ export default {
             if (state.chart) state.chart.dispose();
         });
 
-        (() => {
-            getChartData();
-        })();
+        watch(() => props.period, () => {
+            getData();
+        });
+
+        getData();
```

Comment:
```suggestion
        watch(() => props.period, () => {
            getData();
        }, { immediate: true });
```

이렇게 하면 조금 더 간결해질지는 대신, init이 watch 내에 있어 헷갈릴 수도 있을 듯 하네요. 
항상 고민되는 문제 ㅎㅎ 어떻게 생각하세요?

Reply:
이렇게 초기화 하나만 하는 간단한 경우에는 true로 줘서 하는게 좋아보이네요!!!
초기화함수가 따로 있는 경우에는 구분하는게 덜 헷갈릴거같아유!

Reply:
수정했습니당~~!